vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/validation/MultiSelectComboBoxBinderValidationIT.java

```diff
@@ -36,49 +36,60 @@ public void required_changeValue_assertValidity() {
 
         // Binder validation fails
         testField.selectByText("bar");
+        assertValidationCount(1);
         assertServerInvalid();
         assertClientInvalid();
         assertErrorMessage(UNEXPECTED_VALUE_ERROR_MESSAGE);
 
         // Binder validation passes
         testField.deselectAll();
+        assertValidationCount(1);
         testField.selectByText("foo");
+        assertValidationCount(1);
         assertServerValid();
         assertClientValid();
 
         // Required fails
         testField.deselectAll();
+        assertValidationCount(1);
         assertServerInvalid();
         assertClientInvalid();
         assertErrorMessage(REQUIRED_ERROR_MESSAGE);
+    }
+
+    @Test
+    public void required_enterCustomValue_assertValidity() {
+        $("input").id(EXPECTED_VALUE_INPUT).sendKeys("foo", Keys.ENTER);
 
-        // Try enter custom value, required fails
-        testField.deselectAll();
         testField.sendKeys("custom", Keys.TAB);
```

Comment:
**note:** There is no validation because the `combo-box` web component reverts custom values when they are disabled.